SparkPost/2web2ui
# FE-871: Empty State hibana wrapper

### What Changed
 - Added EmptyState hibana wrapper.
 - Prettier changes.

### How To Test
 - Enable Hibana (or use appteam 105 staging).
 - Go to a url that doesn't exist, IE: /foo.
 - Check that everything looks alright.

### Note
 - Matchbox has not updated empty state yet but we can fix formatting later.
